src/components/PersonElement.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { Person } from '../types/Person';
+
+interface Props {
+  person: Person,
+  handleSelection: (name: string | undefined, slug: string,) => void,
```

Comment:
It's better to use null instead of undefined. In terms of best practices, it is generally recommended to use undefined when a variable is truly undefined or has not been assigned a value yet. Using null is more appropriate when you want to explicitly indicate that a variable or object property should have no value.